### PR TITLE
Update pal_class_config.lua

### PR DESCRIPTION
**Pull request by Werkfakter**
Added a group check to prevent errors when used on a solo paladin

### CODE REVIEW

File: class_configs/pal_class_config.lua [hunk 1]
```diff
@@ -655,32 +655,41 @@ return {
             {
                 name = "WaveHeal",
                 type = "Spell",
-                cond = function(self, _) return mq.TLO.Group.Injured(RGMercUtils.GetSetting('GroupHealPoint'))() > RGMercUtils.GetSetting('GroupInjureCnt') end,
+                active_cond = function(self, aaName) return RGMercUtils.AAReady(aaName) end,
+                cond = function(self, aaName)
+                    if mq.TLO.Group() then return mq.TLO.Group.Injured(RGMercUtils.GetSetting('GroupHealPoint'))() > RGMercUtils.GetSetting('GroupInjureCnt') end
```
Comment (DerpleMQ2) [inline class_configs/pal_class_config.lua:660]: these should all have an else return false.

File: class_configs/pal_class_config.lua [hunk 2]
```diff
@@ -655,32 +655,41 @@ return {
...
+                end,
             },
             {
                 name = "WaveHeal2",
                 type = "Spell",
-                cond = function(self, _) return mq.TLO.Group.Injured(RGMercUtils.GetSetting('GroupHealPoint'))() > RGMercUtils.GetSetting('GroupInjureCnt') end,
+                active_cond = function(self, aaName) return RGMercUtils.AAReady(aaName) end,
+                cond = function(self, aaName)
+                    if mq.TLO.Group() then return mq.TLO.Group.Injured(RGMercUtils.GetSetting('GroupHealPoint'))() > RGMercUtils.GetSetting('GroupInjureCnt') end
```
Comment (DerpleMQ2) [inline class_configs/pal_class_config.lua:668]: these should all have an else return false.

File: class_configs/pal_class_config.lua [hunk 3]
```diff
@@ -655,32 +655,41 @@ return {
...
+                end,
             },
             {
                 name = "Aurora",
                 type = "Spell",
-                cond = function(self, _) return mq.TLO.Group.Injured(RGMercUtils.GetSetting('GroupHealPoint'))() > RGMercUtils.GetSetting('GroupInjureCnt') end,
+                active_cond = function(self, aaName) return RGMercUtils.AAReady(aaName) end,
+                cond = function(self, aaName)
+                    if mq.TLO.Group() then return mq.TLO.Group.Injured(RGMercUtils.GetSetting('GroupHealPoint'))() > RGMercUtils.GetSetting('GroupInjureCnt') end
```
Comment (DerpleMQ2) [inline class_configs/pal_class_config.lua:676]: these should all have an else return false.

File: class_configs/pal_class_config.lua [hunk 4]
```diff
@@ -655,32 +655,41 @@ return {
...
+                end,
             },
             {
                 name = "Gift of Life",
                 type = "AA",
+                active_cond = function(self, aaName) return RGMercUtils.AAReady(aaName) end,
                 cond = function(self, aaName)
-                    return mq.TLO.Group.Injured(RGMercUtils.GetSetting('GroupHealPoint'))() > RGMercUtils.GetSetting('GroupInjureCnt') and
-                        RGMercUtils.AAReady(aaName)
+                    if mq.TLO.Group() then return mq.TLO.Group.Injured(RGMercUtils.GetSetting('GroupHealPoint'))() > RGMercUtils.GetSetting('GroupInjureCnt') end
```
Comment (DerpleMQ2) [inline class_configs/pal_class_config.lua:684]: these should all have an else return false.

File: class_configs/pal_class_config.lua [hunk 5]
```diff
@@ -655,32 +655,41 @@ return {
...
                 end,
             },
             {
                 name = "Hand of Piety",
                 type = "AA",
+                active_cond = function(self, aaName) return RGMercUtils.AAReady(aaName) end,
                 cond = function(self, aaName)
-                    return mq.TLO.Group.Injured(RGMercUtils.GetSetting('GroupHealPoint'))() > RGMercUtils.GetSetting('GroupInjureCnt') and
-                        RGMercUtils.AAReady(aaName)
+                    if mq.TLO.Group() then return mq.TLO.Group.Injured(RGMercUtils.GetSetting('GroupHealPoint'))() > RGMercUtils.GetSetting('GroupInjureCnt') end
```
Comment (DerpleMQ2) [inline class_configs/pal_class_config.lua:692]: these should all have an else return false.

File: class_configs/pal_class_config.lua [hunk 6]
```diff
@@ -655,32 +655,41 @@ return {
             {
                 name = "WaveHeal",
                 type = "Spell",
-                cond = function(self, _) return mq.TLO.Group.Injured(RGMercUtils.GetSetting('GroupHealPoint'))() > RGMercUtils.GetSetting('GroupInjureCnt') end,
+                active_cond = function(self, aaName) return RGMercUtils.AAReady(aaName) end,
```
Comment (DerpleMQ2) [inline class_configs/pal_class_config.lua:658]: active_cond is meant to describe when something is active so this is probably wrong as it is true if the AA is ready not if it is currently on you.

**Review by DerpleMQ2 — CHANGES_REQUESTED** [review]
See comments

File: class_configs/pal_class_config.lua (hunk 6 above)
Comment (Werkfakter) [reply class_configs/pal_class_config.lua:658]: Understood. Removed the active_cond

File: class_configs/pal_class_config.lua (hunk 5 above)
Comment (Werkfakter) [reply class_configs/pal_class_config.lua:692]: Added

File: class_configs/pal_class_config.lua (hunk 4 above)
Comment (Werkfakter) [reply class_configs/pal_class_config.lua:684]: Added

File: class_configs/pal_class_config.lua (hunk 3 above)
Comment (Werkfakter) [reply class_configs/pal_class_config.lua:676]: Added

File: class_configs/pal_class_config.lua (hunk 2 above)
Comment (Werkfakter) [reply class_configs/pal_class_config.lua:668]: Added

File: class_configs/pal_class_config.lua (hunk 1 above)
Comment (Werkfakter) [reply class_configs/pal_class_config.lua:660]: Added

File: class_configs/pal_class_config.lua
Comment (Werkfakter) [inline class_configs/pal_class_config.lua:1]: Recasts DPU on cooldown, even if all buffs still up. Trying to figure out a way to only cast if missing any of the buffs

**Review by Werkfakter — COMMENTED** [review]
Removed active_cond checks and added a return false. Trying to find a way to only cast DPU if the buffs are missing

File: class_configs/pal_class_config.lua (hunk 1 above)
Comment (DerpleMQ2) [reply class_configs/pal_class_config.lua:660]: i think you need to push your commit.  i still only see 1 commit in this PR.

**Pull request closed without merging**